pkg/cli/version.go

```diff
@@ -0,0 +1,22 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/acorn-io/acorn/pkg/version"
+	"github.com/spf13/cobra"
+)
+
+func NewVersion(c CommandContext) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "version",
+		Short:   "Version information for acorn",
+		Example: "acorn version",
+		Run: func(cmd *cobra.Command, args []string) {
+			fmt.Printf("acorn version %s\n", version.Get().String())
+		},
+		ValidArgsFunction: newCompletion(c.ClientFactory, appsCompletion).withShouldCompleteOptions(onlyNumArgs(1)).complete,
+	}
```

Comment:
didn't notice this the first time, but it looks like this is adding completion from the `apps` command to `acorn version`.

@imrajdas Can you change this to accept no args?

```suggestion
	cmd := &cobra.Command{
		Use:     "version",
		Short:   "Version information for acorn",
		Example: "acorn version",
		Run: func(cmd *cobra.Command, args []string) {
			fmt.Printf("acorn version %s\n", version.Get().String())
		},
		Args: cobra.NoArgs,
	}
```

Should be good to merge afterwards.

Reply:
done